app/lib/state_change_notifier.rb

```diff
@@ -1,5 +1,14 @@
 class StateChangeNotifier
-  def self.call(event, candidate: nil, application_choice: nil, application_form: nil)
+  def self.sign_up(candidate)
+    helpers = Rails.application.routes.url_helpers
+    candidate_number = Candidate.where(hide_in_reporting: false).count
+    text = ":sparkles: Candidate ##{candidate_number} just signed up"
```

Comment:
```suggestion
    text = ":sparkles: #{candidate_number.ordinalize} sign up"
```
What you've suggested is definitely an improvement, but developers may visually interpret hash sign + number as the active record id. Using `ordinalize` could address this, as it makes it look less like an id.

Reply:
Maybe? e.g. `The 987th candidate has signed up`

```ruby
text = ":sparkles: The #{candidate_number.ordinalize} candidate has signed up"
```